### DISCUSSION TITLE
为什么 会有人举报关于 PCL 的各种服务？

### DISCUSSION THREAD

**Discussion by gytxtx** [Meloong-Git/PCL#3206] [讨论]
![image](https://github.com/Hex-Dragon/PCL2/assets/62281844/eacd6a4f-1355-4179-9bb6-e943e1b9a051)

[相关动态](https://www.bilibili.com/opus/889422052683939862)

**Comment by allMagicNB** [comment]
谁跟你说MCBBS是因为举报才维护的了？

**Reply by burningtnt** [reply]
谁知道呢…… MCBBS 到现在也没给出详细的维护理由

**Reply by allMagicNB** [reply]
> 谁知道呢…… MCBBS 到现在也没给出详细的维护理由

那粗略地说就是系统维护什么的喽（

**Reply by gytxtx** [reply]
我不好说了

**Comment by allMagicNB** [comment]
反正要么是跟龙猫有仇的举报的，要么就是TZG事件还在继续……？

**Reply by gytxtx** [reply]
🤔

**Reply by ghost** [reply]
很有可能是TZG

**Reply by WForst-Breeze** [reply]
没有证据不能下结论😶容易误解

**Reply by ghost** [reply]
不过国家反扎中心是咋回事，封多少次了，连皮肤API都封了

**Reply by allMagicNB** [reply]
> 不过国家反扎中心是咋回事，封多少次了，连皮肤API都封了

《反扎中心》

**Comment by WForst-Breeze** [comment]
反对 PCL 并且针对其服务进行攻击的大部分无非就是抓住报毒和赞助机制这两点来作为理由，当然也有人就单纯是看不得别人好而已

（当然也可能有其他理由，大同小异吧，毕竟 PCL 这一路下来经历了不少事情，肯定多少会有一些这样的人存在的）

**Reply by allMagicNB** [reply]
举例子：TZG（

**Comment by iPodtouch6557** [comment]
但凡涉及钱就是屁事一大堆
（我不是反对赞助体制，只是说这个道理）

**Comment by iPodtouch6557** [comment]
我发现百度百科部分词条的链接也被举报了
